eng/pipelines/templates/jobs/build.yml

```diff
@@ -78,14 +78,17 @@ jobs:
 
   - task: Powershell@2
     displayName: "(Native AOT) Install Linux arm64 cross compile toolchain"
-    condition: and(succeeded(), eq('${{ parameters.OSName }}', 'linux'), eq(variables['Architecture'], 'arm64'))
+    # The arm64 cross-compilation toolchain install on Linux x64 has glitches. This will be re-enabled once resolved.
+    # condition: and(succeeded(), eq('${{ parameters.OSName }}', 'linux'), eq(variables['Architecture'], 'arm64'))
+    condition: eq('false', 'true')
```

Comment:
nit: I would have just commented out the entire step as opposed to doctoring the condition.

Reply:
I see. thanks Wes. I just pushed a pr to re-enable the checks for arm64.